src/kubernetes_cleanup_pods_by_name/kubernetes_cleanup_pods_by_name/config.py

```diff
@@ -0,0 +1,79 @@
+import os
+import argparse
+
+# add /src to sys.path for importing modules in 'lib' dir
+# an alternative to having to add context.py to every module
+# https://stackoverflow.com/a/4383597
+import sys
+sys.path.insert(1, os.path.abspath('src'))
```

Comment:
This is a proposal/alternative for absolving the need to add `context.py` to each module. Seems to work fine but I may be overlooking something so let me know what you think.

Reply:
Thanks for QA and raising the issue. I addressed this by making [the following change](https://github.com/artsy/opstools/pull/29/commits/79d31075eec52e915a3f2778b1a8d61945c0f540) which appears to work now no matter where the script is executed from, including outside the project.

```
➜  code pwd
/Users/ov/code
➜  code opstools/src/kubernetes_cleanup_pods_by_name/cleanup.py foo 12
INFO:root:Cleaning up pods in default namespace that are older than 12 hours and contain 'foo' in their name
INFO:root:Done.
➜  code cd opstools    
➜  opstools ✗ src/kubernetes_cleanup_pods_by_name/cleanup.py foo 12         
INFO:root:Cleaning up pods in default namespace that are older than 12 hours and contain 'foo' in their name
INFO:root:Done.
➜  opstools ✗ cd src 
➜  src ✗ kubernetes_cleanup_pods_by_name/cleanup.py foo 12    
INFO:root:Cleaning up pods in default namespace that are older than 12 hours and contain 'foo' in their name
INFO:root:Done.

```

Still proposing to avoid adding `context.py` just to load `lib` unless there's a plan to use the file for other things.

Reply:
Cool. Looks like the `__file__ + '/../../..'` is doing essentially a similar thing as [Path(__file__).parents[2]](https://github.com/artsy/opstools/blob/main/src/kubernetes_cleanup_namespaces/kubernetes_cleanup_namespaces/context.py#L7). Up to you on whether to use `pathlib` or not.

About whether to have this done in `config.py` or `context.py`, we might want to consider `pods_by_name.py`. The [lib import there](https://github.com/artsy/opstools/pull/29/files#diff-4dbf790be19fd7d3b2b9185e64876a929b0b3704ded4916fa0982b627acd26b5R3) works only because of the `__file__ + '/../../..'` done in `config.py`. If a person is looking at `pods_by_name.py` only, it might be a bit mysterious as to why the import works, or if someone is writing a new script and copying from `pods_by_name.py` as an example, the person would find the import to not work and spend some time troubleshooting.

If we accept the convention that a `config.py` is required (even for `import`s in all the files to work), then your approach seems fine.

Reply:
Thanks for expanding on this and providing possible scenarios. Will remove this and add `context.py` back in.